src/core/agent/services/signifyNotificationService.ts

```diff
@@ -236,10 +236,10 @@ class SignifyNotificationService extends AgentService {
 
   async onSignifyOperationStateChanged(
     callback: ({
-      oid,
+      recordId,
```

Comment:
can we keep this variable named `oid`? To match KERIA. Otherwise everything looks good